app/src/main/java/org/smartregister/addo/listeners/AddoBottomNavigationListener.java

```diff
@@ -25,29 +25,15 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
         BaseRegisterActivity baseRegisterActivity = (BaseRegisterActivity) context;
 
         if (item.getItemId() == org.smartregister.family.R.id.action_family) {
-            if (context instanceof FamilyRegisterActivity) {
+            if (context instanceof AddoHomeActivity) {
                 baseRegisterActivity.switchToBaseFragment();
             } else {
-                Intent intent = new Intent(context, FamilyRegisterActivity.class);
+                Intent intent = new Intent(context, AddoHomeActivity.class);
                 context.startActivity(intent);
                 context.finish();
             }
```

Comment:
All the two options technically takes the user to the same place if am not mistaken, is it?